x-pack/plugins/endpoint/server/routes/alerts/details/lib/pagination.ts

```diff
@@ -39,11 +34,11 @@ export class AlertDetailsPagination extends Pagination<
   ): Promise<SearchResponse<AlertEvent>> {
     const reqData: AlertSearchQuery = {
       pageSize: 1,
-      sort: EndpointAppConstants.ALERT_LIST_DEFAULT_SORT,
```

Comment:
Why dis?

Reply:
can restore it. the value in the constants class was not typed right so i just used the literal

Reply:
restored it

Reply:
restored it